test/integration/voldemort/performance/benchmark/Benchmark.java

```diff
@@ -642,6 +651,18 @@ public static void main(String args[]) throws IOException {
               .withRequiredArg()
               .describedAs(LOCAL_SERVER_PROPERTIES)
               .ofType(String.class);
+        parser.accepts(WORKLOAD_CLASS, "workload class (full name)")
+              .withRequiredArg()
+              .describedAs(WORKLOAD_CLASS)
+              .ofType(String.class);
```

Comment:
So you don't want to allow a user to implement her own workload class? I thought this might be more flexible. For example, a user may simulate the time interval of requests sent based on traces.

I can certainly use workload-type and give it two options like default-workload and trace-workload.